src/features/Chat/components/ActiveWindow/Dialogs/ReportDialog.tsx

```diff
@@ -0,0 +1,147 @@
+import { selectUserId } from '@/features/Authentication/userSlice';
+import { useAppSelector } from '@/store';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import styled from 'styled-components';
+import { IconButton, TextButton } from '@/components/Buttons';
+import LabeledInput from '@/components/LabeledInput';
+import { palette } from '@/components/variables';
+import ReportSuccessDialog from './ReportSuccessDialog';
+import Text from '@/components/Text';
```

Comment:
Can you group the imports by type and order them, like in some othe files?

- react-stuff
- redux-stuff
- utils
- components

Reply:
Ah yes, nice catch!

Reply:
Fixed!